dotcom-rendering/src/model/buildLightboxImages.ts

```diff
@@ -78,5 +78,5 @@ export const buildLightboxImages = (
 			alreadySeen.push(imageId);
 		}
 	});
-	return uniqueImages;
+	return uniqueImages.filter(image => image.role !== 'thumbnail');
```

Comment:
I don’t think that’s correct (== I actually know it’s not). We should not exclude thumbnails (whether icons, even nicely smaller, are too big on them is an unrelated matter). In-article images set to `thumbnail` weighting (_role_ in your world) are just normal images, high resolution too and they should be included. See eg. example [in here](https://www.theguardian.com/artanddesign/2023/feb/08/kareem-philadelphia-horseman-cowboy-idris-elba-cian-oba-smiths-best-photograph).

This is also important for parity with Live apps Lightbox (where they re shown). Also, it’s parity with frontend.

I know that excluding <620px (esp. as it’s width-only) looks arbitrary and daft. But I think it’s correct now for at least those reasons:
- parity with frontend
- parity with Live apps (possibly Editions too, can’t remember)
- prevents “furniture” quite successfully
- having control over this behaviour in Composer comes with it’s own set of potential issues, even forgetting the amount of work needed to introduce new field to the whole pipeline and frontends

This isn’t to say it should stay like this forever. But I think having parity and knowing “furniture” assets have been specifically created to be always below 620px threshold over the years warrants it could stay like this for now. Excluding `thumbnail` will actually be more arbitrary.